FirebaseInAppMessaging/Tests/Unit/FIRIAMDisplayExecutorTests.m

```diff
@@ -599,8 +599,12 @@ - (void)testAnalyticsTrackingOnMessageClickCase {
 
   [self.clientMessageCache setMessageData:@[ self.m2 ]];
 
```

Comment:
this seems like a test for the positive case where a buttion with an action *does* log. Do we also have a test for a button without an action *not* logging?

Reply:
Added this in 8d26898.